Build failure with Conduit

When I try directly following the Spack build instructions (commit 3cddaf212a3dd6be3c9f1016edfcdf7bf101c3cb, Pascal), I get the following error:
```
/usr/WS1/moon13/src/lbann/model_zoo/jag_utils/generate_corrupt_samples.cpp:33:10: fatal error: conduit_relay_io_handle.hpp: No such file or directory
 #include "conduit_relay_io_handle.hpp"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

I don't see this problem when I use `scripts/build_lbann_lc.sh`, so all signs point to something with Conduit.

--- a/model_zoo/jag_utils/generate_corrupt_samples.cpp
+++ b/model_zoo/jag_utils/generate_corrupt_samples.cpp
@@ -30,7 +30,7 @@
 #ifdef LBANN_HAS_CONDUIT
 
 #include "conduit/conduit.hpp"
-#include "conduit_relay_io_handle.hpp"
+#include "conduit/conduit_relay_io_handle.hpp"
 #include <iostream>
 #include <fstream>
 #include <vector>

